src/components/v5/shared/Navigation/Sidebar/sidebars/AccountPageSidebar/AccountPageSidebar.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  CreditCard,
+  Faders,
+  UserCircleGear,
+  Wrench,
+} from '@phosphor-icons/react';
+import React from 'react';
+
+import {
+  USER_ADVANCED_ROUTE,
+  USER_CRYPTO_TO_FIAT_ROUTE,
+  USER_EDIT_PROFILE_ROUTE,
+  USER_PREFERENCES_ROUTE,
+} from '~routes';
+import SidebarRouteItem from '~v5/shared/Navigation/Sidebar/partials/SidebarRouteItem/index.ts';
+import Sidebar from '~v5/shared/Navigation/Sidebar/Sidebar.tsx';
+
+export const AccountPageSidebar = () => {
+  return (
+    <Sidebar
+      className="!w-[216px]"
+      headerClassName="mb-8"
+      colonySwitcherProps={{ showColonySwitcherText: true }}
+    >
+      <SidebarRouteItem
+        path={`/account/${USER_EDIT_PROFILE_ROUTE}`}
+        translation={{ id: 'userProfile.title' }}
+        icon={UserCircleGear}
+      />
+      <SidebarRouteItem
+        path={`/account/${USER_PREFERENCES_ROUTE}`}
+        translation={{ id: 'userPreferencesPage.title' }}
+        icon={Faders}
+      />
+      <SidebarRouteItem
+        path={`/account/${USER_ADVANCED_ROUTE}`}
+        translation={{ id: 'userAdvancedPage.title' }}
```

Comment:
Could you please update i18n titles here :) 

| Figma | Current branch |
| ------ | ------ |
| <img width="231" alt="image" src="https://github.com/user-attachments/assets/b86d6869-4ef8-4b9c-ae0c-05389d8067db"> | <img width="206" alt="image" src="https://github.com/user-attachments/assets/ab32ef27-511c-4438-990b-7fba6c7d8987"> |

Reply:
Oh wow good spot! @Nortsova

Reply:
Pushed @Nortsova ! 🚀

Reply:
Great, thank you! 🙌 Also could you please do last change here? "Advanced -> Advances settings" after that I will approve, thanks!!

Reply:
Pushed! 🚀 @Nortsova